packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -536,5 +536,6 @@ export const messages = {
     'Not able to parse current results. Raw result: %s',
   package_id_validation_error:
     'Package ID should be a 15 or 18 character Id that starts with 04t',
-  package_id_gatherer_placeholder: '04t...'
+  package_id_gatherer_placeholder: '04t...',
+  sobjects_refresh: 'Code completion and other language server features may not work. Run SFDX: Refresh SObject Definitions to update your local cache.'
```

Comment:
> Code completion and other language server features may not work

Maybe we can be more specific about the SObject autocompletion specifically. "To take advantage of autocompletion for SObjects in Apex code, run the `SFDX: Refresh SObject Definitions` command. Click [here](https://developer.salesforce.com/tools/vscode/en/apex/writing/#code-smartness-for-sobjects) to learn more." @hbc-bindu will have the best wording of that though.

Reply:
@jag-sfdc @brpowell - can we use the wording I suggested below for the other message?

You don't have any sObjects cached locally. To update your local cache, run SFDX: Refresh sObject Definitions.

I'm suggesting we go with what I'm seeing as how we spell "sObject".